Javascript/q40.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>q40</title>
+
+    <style>
+      body {
+        font-family: Verdana, Geneva, Tahoma, sans-serif;
+        font-size: small;
+      }
+
+      .flex {
+        display: flex;
+        gap: 4px;
+        flex-wrap: wrap;
+        width: 240px;
+      }
+    </style>
+
+    <script>
+      function onClicking() {
+      const v= document.querySelector("input").value;
+      document.getElementById("demo2").innerHTML= "Oh, hello there" + " " + v;
+    }  
+    </script>
+  </head>
+
+  <body>
+    <div class="flex">
+      <input id="demo" placeholder="Your first name"></input>
+      <button onclick="onClicking()">say hello</button>
+      <div id="demo2"></div>
```

Comment:
`demo2`? The name suggests that it is somehow a similar thing to `demo` above? Thats why you should use proper names. Something like `output-text` would be better

Reply:
ok